common/SQL/CDTCDPH_VACCINE/vaccines_by_race_eth.sql

```diff
@@ -5,15 +5,28 @@
 with
 GB as ( --Master list of corrected data grouped by region/category
   select
-    coalesce(RACE_ETH,'Unknown') "CATEGORY",
+    case RACE_ETH when 'Unable to report due to policy/law' then 'Unknown' else coalesce(RACE_ETH,'Unknown') end "CATEGORY",
+    --coalesce(RACE_ETH,'Unknown') "CATEGORY",
     case when MIXED_COUNTY in
         ('Alameda','Alpine','Amador','Butte','Calaveras','Colusa','Contra Costa','Del Norte','El Dorado','Fresno','Glenn','Humboldt','Imperial','Inyo','Kern','Kings','Lake','Lassen','Los Angeles','Madera','Marin','Mariposa','Mendocino','Merced','Modoc','Mono','Monterey','Napa','Nevada','Orange','Placer','Plumas','Riverside','Sacramento','San Benito','San Bernardino','San Diego','San Francisco','San Joaquin','San Luis Obispo','San Mateo','Santa Barbara','Santa Clara','Santa Cruz','Shasta','Sierra','Siskiyou','Solano','Sonoma','Stanislaus','Sutter','Tehama','Trinity','Tulare','Tuolumne','Ventura','Yolo','Yuba')
     then MIXED_COUNTY else 'Unknown' end "REGION",
     --count(distinct vax_event_id) "ADMIN_COUNT", --For total doses
     count(distinct recip_id) "ADMIN_COUNT", --For total people
     MAX(case when DATE(ADMIN_DATE)>DATE(GETDATE()) then NULL else DATE(ADMIN_DATE) end) "LATEST_ADMIN_DATE"
   from
-    CA_VACCINE.tab_int_test
+    (select 
+     *,
+        replace( 
+            iff(RECIP_ADDRESS_STATE = 'CA',
+                iff(RECIP_ADDRESS_COUNTY ='Unknown' , 
+                   iff(ADMIN_ADDRESS_COUNTY is null,
+                       'Unknown'
+                   ,ADMIN_ADDRESS_COUNTY)
+               ,RECIP_ADDRESS_COUNTY)
+            ,'Outside California')
+        ,' County') 
+       AS Mixed_county
+    from CA_VACCINE_UAT.VW_TAB_INT_ALL) foo
```

Comment:
Awesome work Carter!

Hannah confirmed we don't want to switch to this new view. We should use the old one CA_VACCINE.tab_int_test

Reply:
Correction, looks like we just want to remove _UAT from the Database name. We will need to keep the mixed_county logic because that field doesn't exist in the CA_VACCINE.VW_TAB_INT_ALL

Reply:
Mixed_county logic is in here now, so we don't need to use CA_VACCINE.tab_int_test